### DISCUSSION TITLE
glances 4.0.8 ignores aliases in glances.conf [network] section

### DISCUSSION THREAD

**Discussion by gbonazzoli** [nicolargo/glances#2836] [General]
In `glances` version 4.0.8 we have the following `glances.conf`

```
[global]
# Refresh rate (default is a minimum of 2 seconds)
# Can be overwrite by the -t <sec> option
# It is also possible to overwrite it in each plugin sections
refresh=2

[diskio]
hide=dm.*,loop.*

[network]
vnet2_alias=VEEAM
```

The configuration wants to give a more significant name to the virtual interface `vnet2`.

But `glances` ignores it as you can see from the screenshot...

<img width="964" alt="Screenshot 2024-06-16 at 11 41 11" src="https://github.com/nicolargo/glances/assets/17742862/ebc62f3c-b8c6-4d0b-8629-d7af1ed11bca">

**Comment by nicolargo** [comment]
The syntax has change between Glances 3 and Glances 4 (read https://github.com/nicolargo/glances/wiki/Glances-4.0-Release-Note#refactor-alias-configuration).

So you need to change your configuration to:

```
[global]
# Refresh rate (default is a minimum of 2 seconds)
# Can be overwrite by the -t <sec> option
# It is also possible to overwrite it in each plugin sections
refresh=2

[diskio]
hide=dm.*,loop.*

[network]
alias=vnet2:VEEAM
```

**Reply by gbonazzoli** [reply]
@nicolargo Many thanks !

It works with the new syntax.
I only want to underline that in the online documentation the old syntax is still present:

<img width="1102" alt="Screenshot 2024-06-16 at 21 22 10" src="https://github.com/nicolargo/glances/assets/17742862/7c9c96e5-757c-4cc1-8ec8-e3702a5a9185">
